src/app/visual/colorpicker/colorpicker-visual.component.html

```diff
@@ -1,21 +1,21 @@
 <div id="screenshot-colorpicker">
   <sky-colorpicker #colorPickerExample1>
-    <input
+  <input
```

Comment:
I believe you accidentally removed the tabs in this file.

Reply:
I'll give it a look

Reply:
This has been fixed